buildworker.rst

```diff
@@ -24,18 +24,18 @@ mailing list.  This mailing list is also the place to contact if you want to
 contribute a buildbot but have questions.
 
 As for what kind of buildbot to run...take a look at our `current fleet
-<http://buildbot.python.org/all/buildslaves>`_.  Pretty much anything that isn't
+<http://buildbot.python.org/all/#/builders>`_.  Pretty much anything that isn't
 on that list would be interesting: different Linux/UNIX distributions,
 different versions of the various OSes, other OSes if you or someone are
 prepared to make the test suite actually pass on that new OS.  Even if you only
 want to run an OS that's already on our list there may be utility in setting it
-up: we also need to build and test python under various alternate build
-configurations.  Post to the mailing list and talk about what you'd like to
+up. We also need to build and test python under various alternate build
```

Comment:
I agree that the colon isn't quite right, but I'd rather just replace it with a semicolon than a full stop.  And the space doesn't need to be removed before `Post`.

`</nitpick>`